tests/semanticdb/expect/InventedNames.expect.scala

```diff
@@ -22,10 +22,10 @@ given `* */*<-givens::InventedNames$package.`* *`.*/`: Long/*->scala::Long#*/ =
 given X with
 /*<-givens::InventedNames$package.given_X.*//*->givens::X#*/  def doX/*<-givens::InventedNames$package.given_X.doX().*/ = 7
 
-given (using X/*->givens::X#*/): Y/*->givens::Y#*/ with
+/*->givens::InventedNames$package.given_Y().(x$1)*/given /*<-givens::InventedNames$package.given_Y().*/(using X/*->givens::X#*//*<-givens::InventedNames$package.given_Y().(x$1)*/): Y/*->givens::Y#*/ with
```

Comment:
to fix the usage occurrence of `...given_Y().(x$1)` we need to avoid looking in the rhs of a `Given` method that returns a class with `GivenClass` flag.

The definition occurrences of `...given_Y().` and `...given_Y().(x$1)` are welcomed though.

Reply:
> The definition occurrences of ...given_Y(). and ...given_Y().(x$1) are welcomed though.

There are no actual occurrences in the source code, so in reality they are not useful for us.

Reply:
ok so in this case there should be no occurrence for the definition for `given X with { ... }` either

Reply:
yes, the previous state was exactly what we need.

Reply:
Same with https://github.com/lampepfl/dotty/pull/12979#discussion_r689259730

I tried rebasing onto master to re-generated expected files, and now I see: 

```scala
given (using X/*->givens::X#*/): Y/*->givens::Y#*/ with
  def doY/*<-givens::InventedNames$package.given_Y#doY().*/ = "7"

given [T/*<-givens::InventedNames$package.given_Z_T#[T]*//*<-givens::InventedNames$package.given_Z_T().[T]*/]: Z/*->givens::Z#*/[T/*->givens::InventedNames$package.given_Z_T#[T]*/] with
  def doZ/*<-givens::InventedNames$package.given_Z_T#doZ().*/: List/*->scala::package.List#*/[T/*->givens::InventedNames$package.given_Z_T#[T]*/] = Nil/*->scala::package.Nil.*/
```

Reply:
fixed by https://github.com/lampepfl/dotty/pull/12979/commits/ec3a21a0065799d4d9d2eb11452330ff5ee556ae